examples/common/utility/utility.hpp

```diff
@@ -20,6 +20,7 @@
 #include <cassert>
 #include <cstring>
 #include <cstdlib>
+#include <cmath>
```

Comment:
Probably need to also include `oneapi/tbb/tick_count.h`?

Reply:
The tbb header files are not supposed to be added to utility.hpp. Instead of using tbb_tick_counts I have used std::chrono::time_point